models/user.go

```diff
@@ -119,6 +121,11 @@ func (u *User) BeforeUpdate() {
 	u.UpdatedUnix = time.Now().Unix()
 }
 
+// Set time to last login
```

Comment:
This comment has to start with the function-name, e.g `// SetLastLogin sets the time of last login`. May sound stupid, but `go fmt` requires it :unamused: